components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/deploy-file-dialog.js

```diff
@@ -0,0 +1,461 @@
+                    '</div>' +
+                    '<div class="toggle-divider"/>' +
+                    '<div class="credentials-password">' + inMemoryList[i].password +
+                    '</div>' +
+                    '<div class="toggle-divider"/>' +
+                    '<button id="' + i + '" type="button" class="delete-button" ' +
+                    'onclick="deleteServer(\'' + i + '\')">' +
+                    '</button>' +
+                    '</div>' +
+                    '<div class="row-divider">' +
+                    '</div>');
+            }
+            serverContainer.html(serverListHtml);
+        };
+
+        selectAllServers = function (source) {
+            var checkboxes = document.getElementsByName("server-credentials");
+            for (var i = 0, n = checkboxes.length; i < n; i++) {
+                checkboxes[i].checked = source.checked;
+            }
+        };
+
+        viewServerList = function () {
+            var serverList = '<div>' +
+                '</div>';
+            if (inMemoryList.length == 0) {
+                serverList = serverList + "<div id='add-server-alert' " +
+                    "class='add-server-alert'>Add one or more servers" +
+                    "</div>";
+            } else {
+                serverList = serverList +
+                    "<div class='select-boxes' id='select-boxes'>" +
+                    "<input type='checkbox' id='select-all-servers' name='select-all-servers' " +
+                    "onclick='selectAllServers(this)'>" +
+                    "<div class='divider'/>Select All" +
+                    "</div>";
+            }
+            for (var i = 0; i < inMemoryList.length; i++) {
+                serverList = serverList + '<div class="server-block" id="server-block">' +
+                    '<input type="checkbox"git  name="server-credentials" id="check' + i + '" ' +
+                    'class="server-credentials">' +
+                    '<div class="toggle-divider"/>' +
+                    '<div class="host">' + inMemoryList[i].host + ' : ' + inMemoryList[i].port +
+                    '</div>' +
+                    '<div class="toggle-divider"/>' +
+                    '<div class="credentials-username">' + inMemoryList[i].username +
+                    '</div>' +
+                    '<div class="toggle-divider"/>' +
+                    '<div class="credentials-password">' + inMemoryList[i].password +
+                    '</div>' +
+                    '<div class="toggle-divider"/>' +
+                    '<button id="' + i + '" type="button" class="delete-button" ' +
+                    'onclick="deleteServer(\'' + i + '\')">' +
+                    '</button>' +
+                    '</div>' +
+                    '<div class="row-divider">' +
+                    '</div>';
+            }
+            return serverList
+        };
+
+        var DeployFileDialog = Backbone.View.extend(
```

Comment:
Can we remove backbone here and use normal java script initialization. Please refer design view js

Reply:
Ok i'll try